internal/server/middleware.go

```diff
@@ -163,7 +163,8 @@ func orgRequired() gin.HandlerFunc {
 
 func setOrganizationInCtx(srv *Server) gin.HandlerFunc {
 	return func(c *gin.Context) {
-		org, err := getOrgFromRequest(c.Request, srv.dataDB)
+		// TODO: use a transaction
```

Comment:
The middleware changes that merged yesterday make this difficult once again. So I guess i'll need to address this in a follow up.